topic/timeseries/dask-weather-data-import.ipynb

```diff
@@ -72,14 +72,41 @@
     "The following data sets need to be processed:\n",
     "- Daily weather data (daily_weather.parquet)\n",
     "- Cities (cities.csv)\n",
-    "- Countries (countries.csv)"
+    "- Countries (countries.csv)\n",
+    "\n",
+    "The subsequent code cell acquires the dataset directly from kaggle.com.\n",
+    "To properly configure the notebook to use corresponding credentials\n",
+    "after signing up on Kaggle, define the `KAGGLE_USERNAME` and\n",
+    "`KAGGLE_KEY` environment variables. Alternatively, put them into the\n",
+    "file `~/.kaggle/kaggle.json` in your home folder, like this:\n",
+    "```json\n",
+    "{\n",
+    "  \"username\": \"acme\",\n",
+    "  \"key\": \"2b1dac2af55caaf1f34df76236fada4a\"\n",
+    "}\n",
+    "```\n",
+    "Another variant is to acquire the dataset files manually, and extract\n",
+    "them into a folder called `DOWNLOAD`. In this case, you can deactivate\n",
+    "those two lines of code, in order to skip automatic dataset acquisition."
```

Comment:
That's the section, now within the notebook itself, which educates readers about Kaggle.